### PR TITLE
cirq 0.15.0

### PR DESCRIPTION
**Pull request by vtomole**

### CODE REVIEW

File: cirq_superstaq/service_test.py [hunk 1]
```diff
@@ -37,17 +37,18 @@ def test_counts_to_results() -> None:
...
         cirq.CNOT(qubits[0], qubits[1]),
-        cirq.measure(qubits[0]),
-        cirq.measure(qubits[1]),
+        cirq.measure(qubits[0], key="0"),
+        cirq.measure(qubits[1], key="1"),
     )
     result = css.service.counts_to_results(
         collections.Counter({"00": 50, "11": 50}), circuit, cirq.ParamResolver({})
     )
+    print(result)
```
Comment (richrines1) [inline cirq_superstaq/service_test.py:51]: stray print

Comment (vtomole) [reply cirq_superstaq/service_test.py]: Removed

**Review by richrines1 — APPROVED** [review]
lgtm

the qasm test won't pass for `cirq==0.14.*`; we should make them less brittle in a future PR so we can run tests with any supported cirq version

**Comment by vtomole** [comment]
> the qasm test won't pass for cirq==0.14.*; we should make them less brittle in a future PR so we can run tests with any supported cirq version

https://github.com/SupertechLabs/cirq-superstaq/issues/283

File: cirq_superstaq/custom_gates.py [hunk 2]
```diff
@@ -588,8 +575,6 @@ def custom_resolver(cirq_type: str) -> Union[Callable[..., cirq.Gate], None]:
         return AceCR
     if cirq_type == "ParallelGates":
         return ParallelGates
-    if cirq_type == "MSGate":
```
Comment (vtomole) [inline cirq_superstaq/custom_gates.py:591]: Link

**Pull request merged**